Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/storage/dao/ContactDiaryLocationVisitDao.kt

```diff
@@ -0,0 +1,23 @@
+package de.rki.coronawarnapp.contactdiary.storage.dao
+
+import androidx.room.Dao
+import androidx.room.Query
+import de.rki.coronawarnapp.contactdiary.storage.entity.ContactDiaryLocationVisitEntity
+import kotlinx.coroutines.flow.Flow
+import org.joda.time.LocalDate
+
+@Dao
+abstract class ContactDiaryLocationVisitDao : BaseRoomDao<ContactDiaryLocationVisitEntity>() {
```

Comment:
what about deleting entries older than 16 days? see chapter Data Retention in the TechSpec

Reply:
yes, Data Retention will be handled later in Subtask #13 (EXPOSUREAPP-4152)